src/modules/LanItems.rb

```diff
@@ -1529,24 +1445,24 @@ def SelectHWMap(hardware)
       # Wireless Card Features
       @wl_auth_modes = Builtins.prepend(
         Convert.convert(
-          Ops.get(hardware, "wl_auth_modes", @nilliststring),
+          Ops.get(hardware, "wl_auth_modes", nil),
```

Comment:
nil is default for get so useless and also convert is useless unless so want to use nil for invalid value. so in general it can look like
`@wl_auth_modes = hardware["wl_auth_modes"] ? ["no-encryption", *hardware["wl_auth_modes"]] : nil`
see explicit nil at the end as from previous code it is not much obvious that it will be nil if wl_auth_modes is nil. BTW can it happen at all? that looks wrong logic for me. as no-encryption should be always supported even if hardware do not specify own modes.